tests/test_templatetags.py

```diff
@@ -94,396 +94,6 @@ def test_nested_component_templates_all_shown_as_used(self):
         self.assertIn("simple_template.html", templates_used)
 
 
-class BlockCompatTests(BaseTestCase):
```

Comment:
Since https://github.com/EmilStenstrom/django-components/pull/859 introduced some tests for compatibility with `{% extends %}` in the file `test_templatetags_extends.py`, I moved also other existing tests there too.